torch/csrc/cuda/Module.cpp

```diff
@@ -1091,6 +1091,9 @@ static void registerCudaPluggableAllocator(PyObject* module) {
   m.def("_cuda_getAllocator", []() {
     return py::cast(torch::cuda::CUDAPluggableAllocator::getCurrentAllocator());
   });
+  m.def("_cuda_host_empty_cache", []() {
```

Comment:
Should this follow the snake case similar to surrounding functions?

Reply:
It'd be great if you took over the PR, I have a testing script in #134332 that reproduces the issue